features/gesplitst-zakelijk-recht.feature

```diff
@@ -0,0 +1,429 @@
+        {
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden?fields=identificatie,type,persoon.omschrijving&type=eigenaar"
+                }
+            },
+            "_embedded": {
+                "zakelijkGerechtigden": [
+                    {
+                        "identificatie": "1002195699",
+                        "type": "eigenaar",
+                        "persoon": {
+                            "omschrijving": "Anne Janssens"
+                        },
+                        "_links": {
+                            "self": {
+                                "href": "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden/1002195699"
+                            }
+                        }
+                    }
+                ]
+            }
+        }
+        """
+
+    Scenario: opvragen van de zakelijk gerechtigden met type van gesplitst recht
+        Als zakelijk gerechtigde wordt gevraagd met "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden?fields=identificatie,type,persoon.omschrijving&type=erfpachter"
+        Dan is het antwoord:
+        """
+        {
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/59020139970000/zakelijkgerechtigden?fields=identificatie,type,persoon.omschrijving&type=erfpachter"
+                },
+                "kadastraalOnroerendeZakenUitSplitsing": {
+                    "href": "/kadastraalonroerendezaken?kadastraleAanduiding=Groningen+P+1399&inclusiefKadastraalOnroerendeZakenUitSplitsing=true"
+                },
+                "kadastraalOnroerendeZakenUitSplitsingInclusiefZakelijkGerechtigden": {
+                    "href": "/kadastraalonroerendezaken?expand=zakelijkGerechtigden&kadastraleAanduiding=Groningen+P+1399&inclusiefKadastraalOnroerendeZakenUitSplitsing=true"
+                }
+            },
+            "_embedded": {},
+            "gesplitstZakelijkRecht": "erfpachter"
+        }
+        """
+
+Rule: bij een onroerende zaak met gesplitst zakelijk recht wordt het gesplitste recht getoond in gesplitstZakelijkRecht
+
+    Scenario: opvragen van een perceel met gesplitst zakelijk recht
+        Als een kadastraal onroerende zaak wordt geraadpleegd met "/kadastraalonroerendezaken/59020139970000?fields=identificatie,type,gesplitstZakelijkRecht"
+        Dan is het antwoord:
+        """
+        {
+            "identificatie": "59020139970000",
+            "domein": "NL.IMKAD.KadastraalObject",
+            "type": "perceel",
+            "gesplitstZakelijkRecht": "erfpachter",
+            "_links": {
+                "self": {
+                    "href": "/kadastraalonroerendezaken/59020139970000?fields=identificatie,type,gesplitstZakelijkRecht"
+                }
+            }
+        }
+        """
+
+    Scenario: opvragen van een appartementsrecht uit gesplitst zakelijk recht
```

Comment:
Deze scenario geldt toch voor elke type KOZ die niet is gesplitst?

Reply:
het scenario wil tonen dat gesplitstZakelijkRecht wordt opgenomen bij een KOZ **die gesplitst is** en niet wordt opgenomen bij een KOZ die is **ontstaan uit splitsing**